### ISSUE TITLE
Archive this Repository

### ISSUE THREAD

**Issue by Mitzi-Laszlo** [solid/community#307]
This repository used to be a hub of information about Solid which can now be found on solidrproject.org which is connected to github.com/solid/solidproject.org to which anyone can make a suggestion as described in https://github.com/solid/process. 

Any objections to archiving this repository?

**Comment by csarven** [comment]
Makes sense. If approved, one of the Administrators should carry this out.

**Comment by Mitzi-Laszlo** [comment]
How should one approve the archival of a Solid GitHub repository?

**Comment by justinwb** [comment]
No objection from me either. I think it's fine to go ahead with archival.

**Comment by marcenuc** [comment]
The correct link is https://solidproject.org/ (notice the 'r' between 'd' and 'p')
